ckan/cli/user.py

```diff
@@ -235,13 +227,13 @@ def list_tokens(username):
                 Strip out microseconds to force formatting as isoformat doesnt
                 have a timespec param on Python 2.
                 """
-                accessed = datetime(
+                accessed = str(datetime(
                     accessed.year,
                     accessed.month,
                     accessed.day,
                     accessed.hour,
                     accessed.minute,
-                    accessed.second).isoformat(u" ")
+                    accessed.second))
```

Comment:
this does not look like what they did here https://github.com/qld-gov-au/ckan/pull/47/files

Reply:
I think you may have the wrong link. But if you mean https://github.com/ckan/ckan/pull/6796 then you are correct, it is different, because what that author tried resulted in a lint error.

[The documentation](https://docs.python.org/2/library/datetime.html#datetime.datetime.__str__) indicates that the `str` value is equivalent to the formatting we want.